course-backend/course-content/apps/api/src/app/question-management/answares/dto/answare.dto.ts

```diff
@@ -0,0 +1,37 @@
+import {Column} from "typeorm";
+import {ApiProperty} from "@nestjs/swagger";
+
+export class AnswareDto{
```

Comment:
**issue:** everything regarding answers exists in an answers folder, you do not need it. Have the answers model, dto, services, etc in the same structure as the questions (no need for an additional folder just for `answers`)